README.md

```diff
@@ -1,3 +1,97 @@
-# Heroku Buildpack: Blackfire Agent
+# Heroku Buildpack: Blackfire
 
-This is the official Heroku Buildpack for Blackfire Agent.
\ No newline at end of file
+> **If your app is written in PHP, you currently don't need to use this buildpack** as it is already [integrated in the
+> official PHP buildpack](https://blackfire.io/docs/integrations/paas/heroku).
+
+This is the official Heroku Buildpack for [Blackfire](https://blackfire.io).
+
+Blackfire is the Code Performance Management solution for developers
+to find and fix performance bottlenecks in dev, test/staging and production.
+
+This buildpack provides **Blackfire Agent** and **CLI Tool**. 
+
+The Agent is the component that aggregates profile data collected by the *probe* from your application engine
+(PHP, Python, Go...), before sending it to Blackfire.io servers so that you can display and analyze it.
+
+The CLI Tool provides a Blackfire client with 2 main commands:
+
+* An HTTP client wrapping cURL to [profile web based apps](https://blackfire.io/docs/cookbooks/profiling-http-via-cli);
+* A client to [profile CLI Commands](https://blackfire.io/docs/cookbooks/profiling-cli).
+
+## Requirements
+
+First and foremost, **you need an account on https://blackfire.io to use this buildpack**.
+
+You need to have **Blackfire probe** installed in your app.
+
+### PHP
+
+If your app is written in PHP, **you currently don't need to use this buildpack as it is already [integrated in the
+official PHP buildpack](https://blackfire.io/docs/integrations/paas/heroku)**.
+
+### Python
+
+Blackfire probe has to be installed via `pip`. As a result, you need to refer `blackfire` as dependency in your
```

Comment:
> Blackfire probe has to be installed via `pip`.
Does that give enough guidance? If I look at our docs, it seems further info would be welcome.

Instead of giving the install procedure here for the Agent, and in our docs for the Probe, shouldn't we have everything in our docs?